scanners/doggo/parser/parser.js

```diff
@@ -0,0 +1,70 @@
+// SPDX-FileCopyrightText: the secureCodeBox authors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+async function parse(fileContent) {
+  const targets = parseResultFile(fileContent);
+
+  if (process.env["DEBUG"] === "true") {
+    console.log("Parsing Result File");
+    console.log(targets);
+  }
+
+  const result = transformToFindings(targets);
+
+  if (process.env["DEBUG"] === "true") {
+    console.log("Transform To Findings");
+    console.log(result);
+  }
+
+  return result;
+}
+
+function transformToFindings(targets) {
+  // Code to transform the scanner results to scb findings
+  return targets.map((target) => {
+    return {
+      name: `DNS Zone: ${target.name} | Type: ${target.type}`,
+      description: `DNS record type "${target.type}" found for zone "${target.name}" with address "${target.address}".`,
```

Comment:
Is "zone" here really the correct / most helpful term?
Aren't these DNS records in a zone?

Reply:
Thx for the hint. Now fixed as discussed 👍🏻